images/win/scripts/Installers/Windows2016/Install-VS2017.ps1

```diff
@@ -97,15 +28,14 @@ if ($instanceFolders -is [array])
     exit 1
 }
 
-$catalogContent = Get-Content -Path ($instanceFolders.FullName + '\catalog.json')
-$catalog = $catalogContent | ConvertFrom-Json
-$version = $catalog.info.id
-$VSInstallRoot = "C:\Program Files (x86)\Microsoft Visual Studio\2017\$ReleaseInPath"
-Write-Host "Visual Studio version ${version} installed"
+$expectedVisualStudioVersion = Get-ToolsetContent | Select-Object -ExpandProperty visualStudio | Select-Object -ExpandProperty version
```

Comment:
`expectedVisualStudioVersion` such naming can confuse in installation file. Let's use just `visualStudioVersion`